compiler/rustc_interface/src/tests.rs

```diff
@@ -640,6 +640,7 @@ fn test_debugging_options_tracking_hash() {
     untracked!(no_interleave_lints, true);
     untracked!(no_leak_check, true);
     untracked!(no_parallel_llvm, true);
+    untracked!(profiler_runtime, None);
```

Comment:
Why did you make this `untracked`?

Reply:
I, perhaps wrongly, thought that if the profiler runtime is changed from one crate to another then the new dependency will necessarily trigger that crate to be built and linked in place of the old one.   But now you've raised it, I guess the new dependency won't even be found?

Reply:
I reverted to tracked.